TestFoundation/TestNSIndexPath.swift

```diff
@@ -277,8 +277,10 @@ class TestNSIndexPath: XCTestCase {
         let ip2: IndexPath = [1, 1, 1]
         
         XCTAssertNotEqual(ip1.hashValue, ip2.hashValue)
-        
-        IndexPath(indexes: [Int.max >> 8, 2, Int.max >> 36]).hashValue // this should not cause an overflow crash
+
+        // this should not cause an overflow crash
+        let hash: Int? = IndexPath(indexes: [Int.max >> 8, 2, Int.max >> 36]).hashValue
```

Comment:
eh the assert below validates the non-crash case; and it is a test of an overflow so I think it should be fine as you have it